rules/instance-metadata-api-access/rule.metadata.json

```diff
@@ -1,24 +1,24 @@
 {
   "name": "instance-metadata-api-access",
   "attributes": {
-    "m$K8sThreatMatrix": "Credential Access::Instance Metadata API",
```

Comment:
Why remove the `K8sThreatMatrix` attribute?

Reply:
Yiscah said she thinks it is not in use, and I didn't see any reference to this attribute in KS so I removed it.

Reply:
I talked to her now. It's better to not remove it now. If we will decide to remove this attribute it will be from all the rules at once

Reply:
done